scripts/tag-as-excluded-promoted-in-rogue.php

```diff
@@ -0,0 +1,195 @@
+<?php
+/**
+ * Script to send more review data to Rogue.
+ * RBs "excluded" here will be tagged with "Hide In Gallery" in Rogue
+ * RBs "promoted" here will be tagged with "Good Photo" in Rogue
+ *
+ * to run:
+ * drush --script-path=../scripts/ php-script tag-as-excluded-promoted-in-rogue.php
+ */
+
+// Pick up from where we left off
+$last_promoted = variable_get('dosomething_rogue_last_post_promoted', NULL);
+$last_excluded = variable_get('dosomething_rogue_last_post_excluded', NULL);
+
+// Get all the promoted posts to send
+if (isset($last_promoted)) {
+	$promoted_posts_in_rogue = db_query("SELECT r.rogue_post_id, rb.uid, rb.nid
+										FROM dosomething_reportback_file f
+										JOIN dosomething_rogue_reportbacks r ON f.fid = r.fid
+										JOIN dosomething_reportback rb ON f.rbid = rb.rbid
+										WHERE f.status = 'promoted'
+										AND r.rogue_post_id > $last_promoted
+										ORDER BY r.rogue_post_id");
+}
+else {
+	$promoted_posts_in_rogue = db_query("SELECT r.rogue_post_id, rb.uid, rb.nid
+										FROM dosomething_reportback_file f
+										JOIN dosomething_rogue_reportbacks r ON f.fid = r.fid
+										JOIN dosomething_reportback rb ON f.rbid = rb.rbid
+										WHERE f.status = 'promoted'
+										ORDER BY r.rogue_post_id");
+}
+
+// Get all the excluded posts to send
+if (isset($last_excluded)) {
+	$excluded_posts_in_rogue = db_query("SELECT r.rogue_post_id, rb.uid, rb.nid
+										FROM dosomething_reportback_file f
+										JOIN dosomething_rogue_reportbacks r ON f.fid = r.fid
+										JOIN dosomething_reportback rb ON f.rbid = rb.rbid
+										WHERE f.status = 'excluded'
+										AND r.rogue_post_id > $last_excluded
+										ORDER BY r.rogue_post_id");
+}
+else {
+	$excluded_posts_in_rogue = db_query("SELECT r.rogue_post_id, rb.uid, rb.nid
+										FROM dosomething_reportback_file f
+										JOIN dosomething_rogue_reportbacks r ON f.fid = r.fid
+										JOIN dosomething_reportback rb ON f.rbid = rb.rbid
+										WHERE f.status = 'excluded'
+										ORDER BY r.rogue_post_id");
+}
+
+$client = dosomething_rogue_client();
+
+// Tag promoted posts
+foreach ($promoted_posts_in_rogue as $promoted) {
```

Comment:
@DFurnes I thought about this, but I wasn't sure if it would be worth it for something that will only be used in this script. Or did you mean have the new method live in this file too?